lib/connection/statement.js

```diff
@@ -824,6 +832,7 @@ function createOnStatementRequestSuccRow(statement, context)
         context.isMulti = true;
         context.multiResultIds = this._resultIds;
         context.multiCurId = 0;
+        context.arrSqlText = context.sqlText.split(';');
```

Comment:
will this work in all cases? do we support store proc creation as multi statement? if yes,  one store proc definition can contains many semicolons. Also how about a comment contains a semi colon?

Reply:
should we always return the whole multistatements for each sub statement?

Reply:
I added code cover the store proc